### PR TITLE
Adds login button styling

### PR DESCRIPTION
**Pull request by MauritsRidder**

### CODE REVIEW

File: src/components/Login/index.js [hunk 1]
```diff
@@ -5,36 +5,65 @@ import {connect} from 'react-redux'
...
-        >Login With Google
-        </button>
-        :
-        <button
-          onClick={() => firebase.logout()}
-        >Log out
-        </button>
-      }
-      <div>
+      <Login>
```
Comment (Randynamic) [inline src/components/Login/index.js]: It’s nice bit all the logic inside the html makes it hard to read.
IMHO it’s better to do it before the return. Like:
Var logic = test ? Value1 : value2;

Return ( <htlm> {logic} </html> )

Sorry for the bad format I’m typing on my phone 😂

Comment (MauritsRidder) [reply src/components/Login/index.js]: Jep, it was a bit messy! Moved it up.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import {firebaseConnect, isLoaded, isEmpty} from 'react-redux-firebase'
+import Lottie from 'react-lottie';
+import * as animationData from "../../pool";
+import styled from 'styled-components'
+import {FadeIn} from 'animate-css-styled-components';
+
+const defaultOptions = {
+  loop: false,
+  autoplay: true,
+  animationData: animationData
```
Comment (stefanlemmen) [inline src/components/Login/Login.js]: you could use the shorthand notation here:
`animationData,`

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  :hover {
+    background-color: #517fa4;
+    transition: all 150ms linear;
+  }
+`;
+
+export const LoginPage = ({firebase, auth}) => {
+  const isLoggedIn = (isLoaded(auth) && !isEmpty(auth));
+
+  let button;
```
Comment (Randynamic) [inline src/components/Login/Login.js]: 😍

**Pull request merged**